ViewPumpSample/build.gradle

```diff
@@ -23,18 +23,20 @@ android {
     }
 
     lintOptions {
-        disable 'AppCompatCustomView'
+        // we don't need to be indexed by firebase and we don't have a custom icon for our sample app
+        disable 'GoogleAppIndexingWarning', 'MissingApplicationIcon'
+        textOutput 'stdout'
+        textReport System.getenv('CI') == 'true'
```

Comment:
I'm not sure if Travis sets this automatically, make sure to add it when we turn it on